config/deploy.rb

```diff
@@ -16,7 +16,7 @@
 set :rails_env, :production
 set :repo_url, 'https://github.com/18F/identity-idp.git'
 set :sidekiq_options, ''
-set :sidekiq_queue, [:mailers, :sms, :analytics]
+set :sidekiq_queue, [:mailers, :sms, :voice, :analytics]
```

Comment:
Good question. We want a separate queue for Voice because of the user experience of receiving phone calls. There is an acceptable delay with SMS messages since you are receiving the entirety of the message at once, however, with voice calls, it's an interactive session which requires the user to be actively listening to a 10 second message. Thus, initiating calls is a higher priority job and having a separate queue will allow us to scale workers appropriately to deliver the UX expected.